src/lucky/page_helpers/text_helpers.cr

```diff
@@ -28,22 +71,54 @@ module Lucky::TextHelpers
     "#{text[0, stop]}#{omission}"
   end
 
+  # Wrap phrases to make them stand out
+  #
+  # This will wrap all the phrases inside a piece of `text` specified by the
+  # `phrases` array. The default is to wrap each with the `<mark>` element.
+  # This can be customized with the `highlighter` argument.
+  #
+  # ```crystal
+  # highlight("Crystal is type-safe and compiled.", phrases: ["type-safe", "compiled"])
+  # ```
+  # outputs:
+  # ```html
+  # Crystal is <mark>type-safe</mark> and <mark>compiled</mark>.
+  # ```
+  #
+  # **With a custom highlighter**
+  #
+  # ```crystal
+  # highlight(
+  #   "You're such a nice and attractive person.",
+  #   phrases: ["nice", "attractive"],
+  #   highlighter: "<strong>\\1</strong>"
+  # )
+  # ```
+  # outputs:
+  # ```html
+  # You're such a <strong>nice</strong> and <strong>attractive</strong> person.
+  # ```
   def highlight(text : String, phrases : Array(String | Regex), highlighter : Proc | String = "<mark>\\1</mark>")
```

Comment:
Actually, could you make the return types `: Nil` for any of the methods that are meant to write HTML directly to the IO so that it won't ever accidentally be used as a `String`? Might also be worth adding a note like: `This method writes HTML directly to the page. It does not return a String` or something like that. Thoughts?